### PR TITLE
Feature no more invalid args

### PR DESCRIPTION
**Pull request by rylandg**
No longer accept invalid args in the CLI. Tested with added spec.

### CODE REVIEW

**Comment by rylandg** [comment]
Fixes https://github.com/binaris/binaris/issues/368

File: cli.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const handleCommand = async function handleCommand(options, specificHandler) {
     msgAndExit(`Invalid subcommand ${cmdSeq[1]} for command ${cmdSeq[0]}`, true);
   }
 
+  console.log(options);
```
Comment (vogre) [inline cli.js]: Forgotten debug print?

**Comment by rylandg** [comment]
@vogre embarassing mistake, thanks for catching it.

**Comment by rylandg** [comment]
Will update.

**Comment by rylandg** [comment]
Agreed. We need both but many can be tested via the spec format.

**Pull request merged**